lib/dialects/abstract/query-generator.js

```diff
@@ -1057,6 +1057,14 @@ class QueryGenerator {
 
         return `(${quotedColumn}->>'${pathStr}')`;
 
+      case 'mariadb':
+        pathStr = `${['$'].concat(paths).join('.')}`;
```

Comment:
_correcting_

Reply:
I appreciate you acknowledging my comments, but please just fix, push and mark as resolved instead, otherwise people get e-mails for every comments :)